README.md

```diff
@@ -1256,6 +1256,63 @@ const response = await client.checkout.shippingMethods(
 )
 ```
 
+### `addPayment`
+
+Creates new Payment for the current checkout.
+
+**Required token:** [Bearer token](#bearer-token) or [Order token](#order-token)
+
+**Parameters schema:**
+
+```ts
+payment_method_id: string
+source_id?: string
+amount?: number
+source_attributes: {
+  gateway_payment_profile_id: string
+  cc_type?: string
+  last_digits?: string
+  month?: string
+  year?: string
+  name: string
+}
+params?: {
+  include?: string
+}
+```
+
+**Success response schema:** [Success schema](#success-schema)
+
+**Failure response schema:** [Error schema](#error-schema)
+
+**Example:**
+
+```ts
+// Logged in user
+const response = await client.checkout.addPayment(
```

Comment:
@aplegatt we need an example for existing credit card, as it's described in Spree API docs